src/Symfony/Component/Amqp/RetryStrategy/RetryStrategyInterface.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace Symfony\Component\Amqp\RetryStrategy;
+
+interface RetryStrategyInterface
```

Comment:
It's used in `NonRetryableException::__construct()` and I added a check in `queue::__construct()`